### DISCUSSION TITLE
New to Github & Slinger

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#204] [Q&A]
Afternoon folks.
As the title says this is day one for me and Github so if this is the wrong place for this don't crucify me- just tell me where to post it.

I've done a lot of reading about Slinger over the past few days and I, like the rest of you, am not best pleased with the powers that be at Slingbox for turning off their servers on November 9th.

From what I have read here it seems that Gerry Dazoo has produced an application that allows users to see the video stream which is output from a Slingbox ( various models supported I understand) via a new SlingServer app running on a Windows machine or a Raspberry Pi and a media player such as VLC.

What I use my Slingbox Pro HD for is viewing either my UK based Virginmedia Tivo V6 or Humax Freesat box while I am abroad so all pretty simple - one Slingbox and one user and one video source (I can live without the Humax Freesat if necessary)

Am I correct in thinking that to replicate this scenario I would need at permanently running server in the UK on the same LAN as the Slingbox. I also believe I would also IDEALLY need a public STATIC IP address which I do NOT have thanks to Virginmedia.

There seems to be a workaround for the static public IP address issue via something provided by Gerry using the Slingbox FinderID but I am very sketchy on how and what this would entail so any more detailed explanation welcome here. 

I can't test anything at the moment because I am not at home in the UK unfortunately.

I would not really want a server running 24/7 in the UK to access my Slingbox so is there a way to access it without running my own Slingserver ? I notice that in the SlingboxServerNetworkingGuide.pdf it talks about a Fully Distributed Configuration ( Cloud Based SS) which is what I assume Slingbox servers currently do for us. Am I right ?  Is this option available currently with Slinger ?

It seems to me that if someone could replicate what Slingbox currently does it would really be the answer to all our prayers. Of course the server(s) would need to be funded and maintained somewhere but I am sure that current Slingbox users would not be averse to spending a few pounds/dollars to support the server(s). Maybe I am being naive or just plain stupid but having to have 100's /1000's of servers running essentially the same code rather than a central server seems not to be the way forward. Feel free to disabuse me of this way of thinking.

Anyway - well done to Gerry and the rest of you guys who have had a hand in creating this alternative for us all. Your time and effort is much appreciated by me at least. Although I spent 30 years in I.T. it was not at the nuts and bolts level you guys deal with so I doubt I can add much value in terms of producing a solution but I am happy to assist in any way I can - maybe testing or documenting for non-technical users.

Steve

**Comment by GerryDazoo** [comment]
The server can run anywhere it does not have to on the local lan with the slingbox and it doesn't have to be running all the time but it is a little more convenient based on your use case. 
No one is going to be able to re-create what sling has done they had hundreds of employees working on this for 10 years.
If you setup a massive cloud server and start selling access I want a cut! But I'm think most people like the idea of Free. There's a facebook group where one of the people has done some Slingbox-Server for dummies. https://www.facebook.com/groups/377795270002137

So, in a nutshell. Run the server code on your laptop when you want to view your slingbox (a simple click on a desktop shortcut) and have it connect to the Internet routeable IP address where the slingbox is in the UK. If your slingbox external IP address changes frequently sign up for a DYNDNS service. There are free ones out there.

**Reply by pfuller88** [reply]
I'll add that many routers have a build in Dynamic DNS updater.   If you can log into your UK router, look for this option and a drop down box with supported DYNDNS services.   You will need to sign up for one in the list and then set the parameters in the router.  If you cannot get into your router, or it does not have a DYNDNS option, then you will need a computer or server running in your UK location to update the IP when it changes, via a DYNDNS service provider and their app.

Cheers
